data/Dockerfiles/postfix/postfix.sh

```diff
@@ -8,24 +8,6 @@ if [[ -z $(grep null /etc/aliases) ]]; then
   newaliases;
 fi
 
-cat <<EOF > /opt/postfix/conf/sql/mysql_relay_recipient_maps.cf
-user = ${DBUSER}
-password = ${DBPASS}
-hosts = mysql
-dbname = ${DBNAME}
-query = SELECT DISTINCT
-  CASE WHEN '%d' IN (
-    SELECT domain FROM domain
-      WHERE relay_all_recipients=1
-        AND domain='%d'
-        AND backupmx=1
-  )
-  THEN '%s' ELSE (
-    SELECT goto FROM alias WHERE address='%s' AND active='1'
-  )
-  END AS result;
-EOF
-
```

Comment:
This query is no longer necessary, since it has been replaced by others.
(see main.cf -> mysql_relay_recipient_maps)